libgstd/gstd_http.c

```diff
@@ -336,15 +337,15 @@ do_request (gpointer data_request, gpointer eval)
   g_free (output);
   output = NULL;
 
-  soup_message_set_response (msg, "application/json", SOUP_MEMORY_COPY,
+  soup_server_message_set_response (msg, "application/json", SOUP_MEMORY_COPY,
       response, strlen (response));
   g_free (response);
   response = NULL;
 
   status = get_status_code (ret);
-  soup_message_set_status (msg, status);
+  soup_server_message_set_status (msg, status, NULL);
   g_mutex_lock (data_request_local->mutex);
-  soup_server_unpause_message (server, msg);
+  soup_server_message_unpause (msg);
```

Comment:
This change was added since libsoup 3.2, for versions minor to 3.2 it will fail as it expect to use soup_server_unpause_message(). There are some distributions that manage libsoup 3.0.7 version like Ubuntu 22.04. For this case, we will need to add conditional logic to support versions < 3.2 and >= 3.2.